BUILDING.md

```diff
@@ -50,6 +50,11 @@ wget -c https://github.com/protocolbuffers/protobuf/releases/download/v3.11.4/pr
 sudo unzip protoc-3.11.4-linux-x86_64.zip -d /usr/local
 ```
 
+When generating source code from `.proto` files, containerd may rely on some external protocol buffer files. These external dependencies should be added to the `/usr/local/include` directory.
+
+> **Note**
+> To install the appropriate version of `protoc` and download any necessary external protocol buffer files, run the install script located at `script/setup/install-protobuf`.
```

Comment:
If we are going to recommend this script, we can remove `wget -c https://github.com/protocolbuffers....` in L49-L50.
If we continue to retain L49-L50, no need to recommend this script.

Reply:
I removed the `wget` command example. It makes more sense to me to point to the script as it will install the correct version of `protoc` and download any necessary external `.proto` files as well.